.github/workflows/test-suite.yml

```diff
@@ -33,47 +33,47 @@ on:
         default: true
 
   schedule:
-    # https://crontab.guru/#0_0_2-31_*_0
-    - cron: "0 0 2-31 * 1-6" # workflow runs at 00:00 on every day-of-month from 2 through 31 and on every day-of-week from Monday through Saturday.
-    - cron: "0 0 2-31 * 0" # workflow runs at 00:00 on every day-of-month from 2 through 31 and on Sunday.
-   
+    - cron: "0 0 2-31 * *" # workflow runs every day at midnight UTC except on the first day of the month
 concurrency:
   group: '${{ github.workflow }}'
   cancel-in-progress: false
    
 jobs:
-  versions:
```

Comment:
nit: we can update the top level comment to clarify this behaviour of running qa on sundays.

Reply:
Added in: 86a23144b77491c400d75ca3289c16a66a92cba3